libherokubuildpack/Cargo.toml

```diff
@@ -18,11 +18,14 @@ all-features = true
 workspace = true
 
 [features]
-default = ["command", "download", "digest", "error", "log", "tar", "toml", "fs", "write", "buildpack_output"]
+default = ["command", "download", "digest", "error", "inventory", "log", "semver", "sha2", "tar", "toml", "fs", "write", "buildpack_output"]
```

Comment:
having `semver` and `sha2` as features made sense when they were in the inventory crate. In the context of the whole `libherokubuildpack` they seem to be worded to broadly. Maybe we can prefix them with `inventory-`?

Reply:
Agree, it makes more sense to prefix them -- I've updated the feature names to `inventory-sha2` and `inventory-sermver`.